README.md

```diff
@@ -28,10 +28,38 @@ Please see [ember-flight-icons/README](ember-flight-icons/README.md). This addon
 
 Please see [flight-icons/README](flight-icons/README.md). This addon can be used in React or Ember apps.
 
+### How to consume an icon in Figma
+
+1. Open your project file within Figma
+2. Navigate to the Assets panel in the left sidebar
+3. Enable the [Flight Icons](https://www.figma.com/file/TLnoT5AYQfy3tZ0H68BgOr/Flight-Icons?node-id=164%3A0) Library  
+_Click the Team library icon (looks like an open book), toggle on Flight Icons_
+4. Search or scroll to find the icon you want to use and drag it into your file
+
+💡 Tip: Swapping instances, sizes, and colors is easy to do from the right sidebar
+
 ## Contributing
 
 See the [Contributing](CONTRIBUTING.md) guide for details.
 
+### How to contribute a new icon in Figma (HashiCorp staff only)
+
+1. [Open a GitHub issue](https://github.com/hashicorp/flight/issues/new) to kick-off work on adding a new icon  
```

Comment:
Opened https://github.com/hashicorp/flight/issues/148 to allow us to better formalize this